src/collection.js

```diff
@@ -217,15 +221,30 @@ export default class Collection {
   }
 
   /**
-   * Retrieve a record by its uuid from the local database.
+   * Resolves a conflict, updating local record according to proposed
+   * resolution — keeping remote record last_modified value as a reference for
+   * further batch sending.
+   *
+   * @param  {Object} conflict   The conflict object.
+   * @param  {Object} resolution The proposed record.
+   * @return {Promise}
+   */
+  resolve(conflict, resolution) {
+    return this.update(Object.assign({}, resolution, {
+      last_modified: conflict.remote.last_modified
+    }));
+  }
```

Comment:
For some reason you now have two versions of this method in the prototype. I recently moved it next to `_handleConflicts`, which should stay there.